source/_patterns/00-atoms/12-images/03-image-ratio-3-2.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Image 3:2
+state: undefined
```

Comment:
Why undefined? Cause we still expand it?

Reply:
Solved via call. :)

Reply:
What was the answer during the call?

Reply:
Undefined is a pattern lab state that we have defined to mark what elements have never been defined by designers.
And I felt, we needed this component.